src/main/webapp/WEB-INF/tags/instructor/results/responsePanel.tag

```diff
@@ -36,7 +36,7 @@
         <ul class="list-group" id="responseCommentTable-${responsePanel.recipientIndex}-${responsePanel.giverIndex}-${responsePanel.qnIndex}"
             style="${not empty responsePanel.comments ? 'margin-top:15px;': 'display:none'}">
             <c:forEach items="${responsePanel.comments}" var="responseComment" varStatus="status">
-                <shared:feedbackResponseComment frc="${responseComment}" firstIndex="${firstIndex}" 
+                <shared:feedbackResponseCommentRow frc="${responseComment}" firstIndex="${firstIndex}" 
                                                 secondIndex="${secondIndex}" thirdIndex="${thirdIndex}" 
                                                 frcIndex="${status.count}"/>
```

Comment:
Move the subsequent parameters three spaces to the right as well so that `secondIndex` and `frcIndex` are both aligned with `frc`.